website/docs/get-started.md

```diff
@@ -28,6 +28,13 @@ table tr:nth-child(2n) {
 
 Or [build from source](#build-from-source). Having trouble? Ask in [Gitter](https://gitter.im/comby-tools/community) or [create an issue on GitHub](https://github.com/comby-tools/comby/issues/new/choose).
 
+To make the command `comby` available to the system, you can create a script in bin containing
+```
+#!/usr/bin/env bash
+docker run --volume $PWD:/pwd --workdir="/pwd" -it comby/comby "$@"
+```
+where `docker` can also be replaced by `podman`.
+
```

Comment:
````suggestion
Tip: If you're using the `comby` docker image, you can add a `comby` command to your terminal by adding a script like this to a directory in your `PATH` (e.g., `/usr/local/bin`):
```bash
#!/usr/bin/env bash
docker run --volume $PWD:/pwd --workdir="/pwd" -it comby/comby "$@"
```
where `docker` can also be replaced by `podman`.
````

Sounds good! I would just make this a bit more accurate that it's specifically for when using the docker image. Will merge ~tomorrow.